Packages/KeyAppKit/Sources/BankTransfer/Models/UserWallet.swift

```diff
@@ -0,0 +1,45 @@
+public struct UserWallet: Codable {
+    public let walletId: String
+    public var accounts: UserAccounts
+}
+
+public struct UserAccounts: Codable {
+    public var eur: UserEURAccount?
+    public var usdc: UserUSDCAccount?
+}
+
+public struct UserEURAccount: Codable {
```

Comment:
do you consider renaming to EURUserAccount? or it's incorrect?

Reply:
i don't mind. i am not sure what is better

Reply:
i thought as we have UserAccounts, and User is like a namespace so it should be first